cmd/update-main.go

```diff
@@ -235,20 +257,22 @@ func getDownloadURL() (downloadURL string) {
 	return minioReleaseURL + "minio"
 }
 
-func getUpdateInfo(timeout time.Duration, mode string) (older time.Duration, downloadURL string, err error) {
-	currentReleaseTime, err := GetCurrentReleaseTime()
+func (u *updateParams) getUpdateInfo(timeout time.Duration) (
```

Comment:
IMO this method needs to be exported ie `GetUpdateInfo()`.  This will give better readability at consumer side.  but its your call.